examples/neo4j/index.js

```diff
@@ -0,0 +1,31 @@
+import { App } from '@tinyhttp/app'
+import { logger } from '@tinyhttp/logger'
```

Comment:
why does it need a logger? 🤔

Reply:
For it to actually work it is not necessary to use it, but I believe it is always good to have the logs so that we can see if the API is up and responding correctly, but if I find it unnecessary I can remove